corehq/motech/repeaters/models.py

```diff
@@ -852,8 +852,6 @@ def wrap(cls, data):
             cls_ = cls.get_class_from_doc_type(data['doc_type'])
             if cls_:
                 return cls_.wrap(data)
-            elif data['doc_type'] == 'Repeater' and settings.UNIT_TESTING:
-                return super(Repeater, cls).wrap(data)
```

Comment:
This change by itself is not quite what I had in mind in https://github.com/dimagi/commcare-hq/pull/30348#discussion_r821062311

In addition to removing it here, I was suggesting that tests should patch this method to do the equivalent to what this was doing. Something like this
```py
@classmethod
def wrap(cls, data):
    if cls.__name__ == "Repeater" and data['doc_type'] == 'Repeater':
        return super(Repeater, cls).wrap(data)
    return Repeater_wrap(data)

Repeater_wrap = Repeater.wrap

@patch.object(Repeater, "wrap", wrap)
class TestRepeaters(TestCase):

    def test_repeater_stuff(self):
        ...
```